fuzzing/src/targets/font-drivers/colrv1.cpp

```diff
@@ -0,0 +1,42 @@
+// colrv1.cpp
+//
+//   Implementation of CffFuzzTarget.
+//
+// Copyright 2021 by
+// Dominik Röttsches, Armin Hasitzka, David Turner, Robert Wilhelm,
+// and Werner Lemberg.
+//
+// This file is part of the FreeType project, and may only be used,
+// modified, and distributed under the terms of the FreeType project
+// license, LICENSE.TXT.  By continuing to use, modify, or distribute
+// this file you indicate that you have read the license and
+// understand and accept it fully.
+
+
+#include "targets/font-drivers/colrv1.h"
+
+#include <utility> // std::move
+
+#include "iterators/faceloaditerator.h"
+#include "iterators/faceprepiterator-bitmaps.h"
+#include "iterators/faceprepiterator-outlines.h"
+#include "visitors/facevisitor-colrv1.h"
+
+  freetype::ColrV1FuzzTarget::
```

Comment:
for consistency's sake: let's please have two empty lines between `#include`s and class definitions

Reply:
Done.